kong-2.4.1-0.rockspec

```diff
@@ -148,6 +148,7 @@ build = {
     ["kong.runloop.plugin_servers"] = "kong/runloop/plugin_servers/init.lua",
     ["kong.runloop.plugin_servers.process"] = "kong/runloop/plugin_servers/process.lua",
     ["kong.runloop.plugin_servers.mp_rpc"] = "kong/runloop/plugin_servers/mp_rpc.lua",
+    ["kong.runloop.plugin_servers.pb_rpc"] = "kong/runloop/plugin_servers/pb_rpc.lua",
```

Comment:
I think a test case in `spec/01-unit/02-rockspec_meta_spec.lua` should have failed here due to the missing entry.